spec/paths/enterprise@{enterprise_id}@content@{content_id}@.yaml

```diff
@@ -0,0 +1,73 @@
+get:
```

Comment:
All paths in content resource is v0 API's ie. of format
`https://{tenant}-api.esper.cloud/api/v0/enterprise/:id/content/`